CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Ewon Flexy Extensions Library Changelog
 
+## Version 1.12.0
+### Features
+- Added additional known status codes in MqttStatusCode class
+- Added SmartMqttManager wrapper of MqttManager to handle common use cases
+- Added SCHttpUtility methods to output response data to a file
+### Bug Fixes
+- Fixed improper Javadoc in AutomaticRetryCodeLinear
```

Comment:
```suggestion
- Fixed improper Javadoc in AutomaticRetryCodeLinear.java
```

Not sure if we should include file extensions but it seems to clarify the fact that we are referencing a filename.

Reply:
Updated